src/cpp/rtps/transport/test_UDPv4Transport.cpp

```diff
@@ -154,7 +154,7 @@ void test_UDPv4Transport::get_ips(
 }
 
 bool test_UDPv4Transport::send(
-        const octet* send_buffer,
+        const std::array<asio::const_buffer, 3>& send_buffer,
         uint32_t send_buffer_size,
```

Comment:
```suggestion
        size_t send_buffer_size,
```

Maybe we should unify the criteria in all the `send` methods. Changing the type to `size_t` where the parameter is a size.

Reply:
I agree with the last sentence

Reply:
Changed in 33959d01d5afabd1b0a76f7df4238b632fca3b57